### ISSUE TITLE
How do we install fish completions (the easiest way)?

### ISSUE THREAD

**Issue by BarbzYHOOL** [fish-shell/fish-shell#9341]
Hello, just wanted to know how to easily install fish completions provided (well hidden) in this directory?
I type "install fish completions" on google and don't find anything useful

**Comment by krobelus** [comment]
fish already comes with lots of completions preinstalled so you should be all set.
Which completions do you wish to install? You can place completion files in `~/.config/fish/completions`.

**Comment by BarbzYHOOL** [comment]
Where are they installed?

I thought maybe that i was missing the flatpak completion because it's so annoying to type the whole flatpak address of a package instead of typing the package name and then it completes the address automatically

**Comment by krobelus** [comment]
it should be somewhere in `ls $fish_complete_path`, specifically `$__fish_data_dir/completions/flatpak.fish`.
Which flatpak command are you trying to tab-complete? Maybe it's a problem with existing completions.

**Comment by robb-randall** [comment]
I have the following completions for flatpak:

```
/usr/share/fish/completions/flatpak.fish
/usr/share/fish/vendor_conf.d/flatpak.fish
/usr/share/fish/vendor_completions.d/flatpak.fish
```

**Comment by BarbzYHOOL** [comment]
ok seems like i have tons of them as well

well i thought i was missing completions because usually fish is super smart, so let's say i'd type `flatpak info Boxy` then press [Tab], it should (in my mind) transform to `flatpak info com.boxy_svg.BoxySVG     ` but it does not, so I always have to type `flatpak list` then find my program and type the correct thing, very annoying

**Comment by krobelus** [comment]
looks like this was a bug in flatpak completions.
Should be fixed by 108108bb5ee61bc5b20717ade6a8e5988a2d21bd, you can grab that file to get the fix right away.
Thanks for reporting.

**Comment by BarbzYHOOL** [comment]
wow thanks krobelus
i'll try it tomorrow
